backend/watcher-service/src/main/kotlin/com/nosql/watcher/repository/WatcherRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.nosql.watcher.repository
+
+import com.nosql.watcher.entity.WatcherEntity
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository
+
+interface WatcherRepository : ReactiveMongoRepository<WatcherEntity, ObjectId>{
+}
```

Comment:
delete curve brackets

Reply:
we will ned them in next task, so it is not critical